packages/image/types/index.d.ts

```diff
@@ -0,0 +1,31 @@
+import { SvelteComponentTyped } from 'svelte';
+import './ambient.js';
+
+export interface GetURL<ProviderSpecificOptions = {}> {
+	(opts: { src: string; width: number; height: number; options?: ProviderSpecificOptions }): string;
+}
+
+export class Image extends SvelteComponentTyped<
+	(
+		| {
+				src: {
```

Comment:
this should be updated to use the types from `vite-imagetools`. I updated the `.svelte` component, but didn't see this one